test/python/circuit/test_circuit_operations.py

```diff
@@ -485,6 +485,36 @@ def test_copy_empty_variables(self):
         self.assertEqual({b, d}, set(copied.iter_captured_vars()))
         self.assertEqual({b}, set(qc.iter_captured_vars()))
 
+    def test_copy_empty_variables_to_captures(self):
```

Comment:
Do you want a test case validating the `alike` mode case?

Reply:
`alike` is the default, so that's tested by the `test_copy_empty_variables` that's immediately above this diff.  I can pass it explicitly in that, if you prefer?

Reply:
Explicit test added in a4a9cea96b.